gymnasium/experimental/wrappers/__init__.py

```diff
@@ -86,3 +86,38 @@
     "VectorRecordEpisodeStatistics",
     "VectorListInfo",
 ]
+
+
+class DeprecatedWrapper(ImportError):
+    """Exception raised when an old version of a wrapper is imported."""
+
+    pass
+
+
+def __getattr__(wrapper_name):
+    """Raises a DeprecatedWrapper exception when an old version of a wrapper is imported, or an ImportError if the wrapper being imported does not exist."""
+    wrapper_module = __name__
+
+    version = wrapper_name[-1]
+    base = wrapper_name[:-1]
+
+    try:
+        version_num = int(version)
+        is_valid_version = True
+    except ValueError:
+        is_valid_version = False
+
+    global_dict = globals()
+
+    if is_valid_version:
+        for act_version_num in range(1000):
+            act_wrapper_name = f"{base}{act_version_num}"
+            if act_wrapper_name in global_dict:
+                if version_num < act_version_num:
+                    raise DeprecatedWrapper(
+                        f"{base}{version_num} is now deprecated, use {act_wrapper_name} instead.\n"
+                        f"To see the changes made, go to "
+                        f"https://gymnasium.farama.org/api/experimental/wrappers/#{wrapper_module}.{act_wrapper_name}."
+                    )
+
```

Comment:
Raise a different error if the version is greater than the wrapper's version.

Reply:
Done